README.md

```diff
@@ -1 +1,69 @@
 # Goi
+
+## Deck Creation DSL
+### Example
+```
+Create Deck French with Tags language, hard, midterm using Style mystyle:  
```

Comment:
would it be easier if we defined things before they're first used. 
e.g. 'mystyle' is called before it's actually implemented, might be simpler if we parsed/implemented it before we called it?

Reply:
Yeah, okay that makes sense